doc/nrf/protocols/matter/end_product/bootloader.rst

```diff
@@ -27,29 +27,30 @@ You can reduce the size of the bootloader image by disabling unnecessary feature
 
+.. _ug_matter_device_bootloader_partition_layout:
+
 Partition layout
 ****************
 
 Each application that uses MCUboot must use :ref:`partition_manager` to define partitions of the flash memory.
 This is needed for the bootloader to know where the current and the new firmware images are located in the flash.
 
-There are multiple ways to define parititions using Partition Manager.
-For example, each :ref:`Matter sample <matter_samples>` provides a :file:`pm_static_dfu.yml` file (one for each configuration) that statically defines the partition layout.
-See :ref:`ug_matter_hw_requirements_layouts` to check the reference partition layout for each supported platform.
-
-Given the size of the Matter stack, it will usually not be possible to fit both the primary and the secondary slot in the internal flash in order to store the current and the new firmware image, respectively.
-Instead, you should use the :ref:`external flash <ug_bootloader_external_flash>` to host the secondary slot.
+Take into account the following points when defining paritions for your end product:
 
-.. note::
-   Remember to enable a proper flash driver when putting the secondary slot in the external flash.
-   For example, if you develop your application on a Nordic Semiconductor's development kit that includes a QSPI NOR flash module, you can do this by setting the :kconfig:option:`CONFIG_NORDIC_QSPI_NOR` Kconfig option.
+* There are multiple ways to define parititions using Partition Manager.
+  For example, each :ref:`Matter sample <matter_samples>` provides a :file:`pm_static_dfu.yml` file (one for each configuration) that statically defines the partition layout.
+  See :ref:`ug_matter_hw_requirements_layouts` to check the reference partition layout for each supported platform.
+* Given the size of the Matter stack, it will usually not be possible to fit both the primary and the secondary slot in the internal flash in order to store the current and the new firmware image, respectively.
+  Instead, you should use the :ref:`external flash <ug_bootloader_external_flash>` to host the secondary slot.
 
-.. note::
-   You should select the partition sizes wisely, as some of them like settings and factory data partitions are not modified during Device Firmware Upgrade (DFU) process.
-   It means that performing Device Firmware Upgrade (DFU) from the one firmware version to another using different parition sizes may not be possible, so you won't be able to change the partition sizes without re-programming the device.
-   Trying to perform DFU between applications using not-compatible partition sizes may result in different behavior depending on which partitions will overlap to anothers.
-   In some cases it may result in specific partition parts corruption and in the other ones it may lead to firmware upgrade failure.
+  .. note::
+      Remember to enable a proper flash driver when putting the secondary slot in the external flash.
```

Comment:
Can we have something else instead of "putting"? It does not sound good to me.

Reply:
@kkasperczyk-no , "placing"?